assets/src/edit-story/components/panels/backgroundColor.js

```diff
@@ -28,32 +28,35 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { InputGroup } from '../form';
 import { SimplePanel } from './panel';
 import getCommonValue from './utils/getCommonValue';
+import BackgroundColorControls from './shared/backgroundColorControls';
 
 function BackgroundColorPanel({ selectedElements, onSetProperties }) {
   const backgroundColor = getCommonValue(selectedElements, 'backgroundColor');
-  const [state, setState] = useState({ backgroundColor });
+  const backgroundOpacity = getCommonValue(
+    selectedElements,
+    'backgroundOpacity'
```

Comment:
I think we should combine `backgroundColor` and `backgroundOpacity` into a single `backgroundColor` as an rgba value. And the same for all other "color" properties. There are lots of different colors out there and pairing their properties in each case could be avoided this way. It could also allow us to composite this feature easier. Thoughts?

/cc @barklund

Reply:
Makes sense.

Reply:
@barklund @dvoytenko Based on the discussion on Slack, will restructuring the colors be handled in a separate issue/PR? 

I started changing the background value to an object of `{r,g,b,a}` but wondering if it makes sense here in case it'll be looked into more seriously elsewhere. Thoughts?

Reply:
I say ignore it as much as possible for now and let's deal with it in #263. We'll do any necessary data migration there too.

Reply:
Will leave it as it is for now to be resolved elsewhere.